src/view/scripts.inc

```diff
@@ -0,0 +1,48 @@
+<!-- Namespace for Skosmos variables -->
+<script>
+  let SKOSMOS = (function () {
+    var content_lang = "{{ request.contentLang }}";
+    var explicitLangCodes = {{ explicit_langcodes ? 'true' : 'false' }};
+    var lang = "{{ request.lang }}";
+    var vocab = "{{ request.vocabid }}";
+    var waypoint_results = {{ parameters ? parameters.searchLimit : NULL }};
+
+    {% if request.page == 'page' and search_results and search_results|length == 1 %}
```

Comment:
Style issue: do we really want to indent the variable declarations that are inside conditional `if` blocks? It means that the rendered HTML/JS will have inconsistent indentation.